pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
     "voila>=0.4,<0.5",
     "pyyaml>=5.4.1",
     "specutils>=1.15",
-    "specreduce>=1.3.0,<1.4.0",
+    "specreduce@git+https://github.com/astropy/specreduce",
```

Comment:
Should release of specreduce and pinning to it block merge here?  Alternatively, we could have the feature dependent on a version check of specreduce without having to bump the min-pin?

Reply:
I'm +1 for release specreduce.